core/bin/block_reverter/src/main.rs

```diff
@@ -33,35 +32,35 @@ enum Command {
     /// Sends revert transaction to L1.
     #[command(name = "send-eth-transaction")]
     SendEthTransaction {
-        /// L1 batch number used to rollback to.
+        /// L1 batch number used to revert to.
```

Comment:
In my head, `rollback` is about the undoing of recent actions as if they never happened.
And `revert` is about undoing itself is recorded as a new action. 
AFAIU here is the case 1( e.g. undoing of recent actions as if they never happened)..so I am confused

Reply:
That's reasonable, but then we'd need to rename the reverter component to something like `BlockRollback` (or `BlockRollbackExecutor`?) because it doesn't do reverts. Would this make more sense to you?

Reply:
>Would this make more sense to you?

Yes. Also, there is a quick solution - I just left the doc with an explanation this part

Reply:
Having [ubiquitous language](https://martinfowler.com/bliki/UbiquitousLanguage.html) (i.e., unified terminology in this case) is kind of important, that's why I've tried to eliminate a mix of "revert" / "rollback"; I think it's worse than using either of these terms consistently.

Reply:
I've retained the name for now, and explained the difference between "roll back" and "revert" in the docs (and it's reflected in method naming).